### PR TITLE
Move daemon files to single directory

### PR DESCRIPTION
**Pull request by tgraf**
Now that the server side code is autogenerated, the daemon directory
can be simplified.
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/mchalla%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/725475%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/292%23issuecomment-282164008%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-02-24T00%3A13%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/725475%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mchalla%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20a1be174e17639b2335b0f5787f5831e29db9f4d2%20daemon/daemon.go%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/292%23discussion_r102850597%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20is%20this%20not%20needed%20anymore%20%3F%22%2C%20%22created_at%22%3A%20%222017-02-24T00%3A00%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/725475%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mchalla%22%7D%7D%2C%20%7B%22body%22%3A%20%22This%20code%20was%20previously%20in%20a%20separate%20directory%20and%20thus%20package.%20The%20log%20variable%20has%20a%20scope%20of%20a%20package%20which%20means%20as%20these%20files%20are%20now%20all%20part%20of%20the%20same%20package%2C%20we%20only%20need%20one%20log%20instance.%20It%20is%20now%20using%20the%20log%20variable%20of%20%60daemon/main.go%60%22%2C%20%22created_at%22%3A%20%222017-02-24T00%3A06%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1417913%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tgraf%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20daemon/daemon.go%3AL48-61%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/cilium/cilium/pull/292%23issuecomment-282164008%22%2C%20%22https%3A//github.com/cilium/cilium/pull/292%23discussion_r102850597%22%2C%20%22https%3A//github.com/cilium/cilium/pull/292%23discussion_r102851395%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/mchalla'></a>

- [x] <a href='#crh-comment-Pull a1be174e17639b2335b0f5787f5831e29db9f4d2 daemon/daemon.go 16'></a> &nbsp;<b><a href='https://github.com/cilium/cilium/pull/292#discussion_r102850597'>File: daemon/daemon.go:L48-61</a></b>
- <a href='https://github.com/mchalla'></a> Why is this not needed anymore ?
- <a href='https://github.com/tgraf'></a> This code was previously in a separate directory and thus package. The log variable has a scope of a package which means as these files are now all part of the same package, we only need one log instance. It is now using the log variable of `daemon/main.go`

<a href='https://www.codereviewhub.com/cilium/cilium/pull/292?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cilium/cilium/pull/292?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cilium/cilium/pull/292?approve=0'></a>&nbsp;<a href='https://github.com/cilium/cilium/pull/292'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by aanm** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cilium/cilium/pull/292'>Refresh</a>.<a href='#crh-update-none'></a>

**Review by aanm — APPROVED** [review]

File: daemon/daemon.go [hunk 1]
```diff
@@ -48,18 +48,13 @@ import (
...
 	k8sRest "k8s.io/client-go/1.5/rest"
 	k8sClientCmd "k8s.io/client-go/1.5/tools/clientcmd"
 	"k8s.io/kubernetes/pkg/registry/core/service/ipallocator"
 )
 
-var (
-	log = logging.MustGetLogger("cilium-net")
-)
-
 // Daemon is the cilium daemon that is in charge of perform all necessary plumbing,
```
Comment (mchalla) [inline daemon/daemon.go:58]: Why is this not needed anymore ?

Comment (tgraf) [reply daemon/daemon.go:58]: This code was previously in a separate directory and thus package. The log variable has a scope of a package which means as these files are now all part of the same package, we only need one log instance. It is now using the log variable of `daemon/main.go`

**Comment by mchalla** [comment]
:shipit:

**Review by mchalla — APPROVED** [review]

**Pull request merged**